packages/vscode-extension/src/builders/customBuild.ts

```diff
@@ -1,28 +1,50 @@
 import path from "path";
 import fs from "fs";
+import os from "os";
+import { mkdtemp } from "fs/promises";
 import { Logger } from "../Logger";
 import { command, lineReader } from "../utilities/subprocess";
 import { CancelToken } from "./cancelToken";
 import { getAppRootFolder } from "../utilities/extensionContext";
+import { extractTarApp } from "./utils";
 
 type Env = Record<string, string> | undefined;
 
+const EXPO_LOCAL_BUILD_PATH_REGEX = new RegExp("You can find the build artifacts in (.*)");
```

Comment:
if we only scan the last line I think we could relax it much more and just take the file paths that are present in the final line of output